auth0_flutter/README.md

```diff
@@ -30,11 +34,27 @@ Auth0 SDK for Android / iOS Flutter apps.
 - [**What is Auth0?**](#what-is-auth0)
 - [**License**](#license)
 
-<!-- 
+## Features
+
+- Web Auth login and logout
+- Automatic storage and renewal of user's credentials
```

Comment:
Not referencing support for custom Credentials Manager, because as of now the `CredentialsManager` abstract class is not exported.